x-pack/plugins/siem/public/components/page/hosts/events_table/index.tsx

```diff
@@ -203,7 +203,8 @@ const getEventsColumns = (
           idPrefix: `host-${pageType}-events-table-${node._id}`,
           render: item => <IPDetailsLink ip={item} />,
         })}
-        :{getOrEmptyTag('source.port', node)}
+        {':'}
+        {getOrEmptyTag('source.port', node)}
```

Comment:
This can be combined into a string interpolation to read better

Reply:
Since this can be a JSX object it should not be as it will show `[Object object]`